pyproject.toml

```diff
@@ -58,18 +58,32 @@ numpy = "1.23.5"
 protobuf = "3.20.3"
 pandas = "2.0.3"
 
-# Needed to have the latest Concrete Python versions
+# Zama sources
 [[tool.poetry.source]]
 name = "zama-pypi-cpu"
 url = "https://pypi.zama.ai/cpu"
 priority = "explicit"
 
-[tool.poetry.dev-dependencies]
+[[tool.poetry.source]]
```

Comment:
not sure about tests, but it could be a dedicated gh action that runs on a machine with gpu and where we install + activate the gpu option from Concrete 

still, I'm not sure adding gpu dependency in this pr makes much sense !